### DISCUSSION TITLE
_partialTextX

### DISCUSSION THREAD

**Discussion by Irfan-Majid** [konvajs/konva#1725] [General]
I cant get _partialTextX after layer.draw(). It is showing previous value

**Comment by lavrton** [comment]
Why do you need it?

**Reply by Irfan-Majid** [reply]
to generate picture in backend i need the value from where text is starting. i am getting scaleX and need the value inbetween scaleX line and text start line.

**Reply by lavrton** [reply]
Can you provide further information? `_partialTextX` is a private property usually shouldn't be used.
